visual-testing-app/src/components/page-content-container-wide/page-content-container-wide.visualroute.tsx

```diff
@@ -60,11 +60,11 @@ export const Component = () => (
           ),
         },
         {
-          name: 'two columns 1/1 with big gap',
-          path: 'two-columns-half-big-gap',
+          name: 'two columns 1/1 with small gap',
```

Comment:
There's another VRT called `two columns 2/1 with big gap` which we should update.
Also, I guess we should use the `l` size for the default two columns VRTs as now they have the big gap by default.

Reply:
Thank you. 

Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2967/commits/787277dd837da085601b9fc31c463bf44c9a3b84